README.md

```diff
@@ -192,3 +191,54 @@ For Unit Testing CppUtest framework is used.
   * Set **BUILD_UNIT_TESTS** option *ON*
   * Set **PATH_CPP_UTEST** Path to the cpputest folder
   * Build and Run the *facemask-plugin-test* project, it will show the report of tests
+  
+  
+  
+## Video Comparison side by side
+
+### Setting up:
+* Install FFmpeg:
+  [ffmpeg](https://www.ffmpeg.org/)
+  **Add a Path** of FFmpeg executable in the Environment Variables:
+  The path should look like: *%PATH_TO_YOUR_DIR%\ffmpeg-%VERSION%\bin*
+  
+### Format Video:
+Formatting video makes start/end of the video detectable by adding a red frame at the end
+- In the *facemask-plugin/data* there is a script: **format.bat**
+- Put your sample video as input.mp4 and click on the format.bat
+- Or call the script from the shell 
+```console 
+./format.bat /path/to/the/video
+```
```

Comment:
I was able to do this when running format.bat from the folder and after putting input.mp4. But I could not do it from command prompt, i get impossible to open for the file. Also, can you add a destination path when calling it from command line?

Reply:
- Fixed it, was a problem with windows standard command line (working in bash I have)
- Added output file parameter